orca-web/src/test/groovy/com/netflix/spinnaker/orca/controllers/TaskControllerSpec.groovy

```diff
@@ -622,8 +623,8 @@ class TaskControllerSpec extends Specification {
        trigger: new DockerTrigger("test-account", "test-repo", "1")
       ]
     ]
-    pipelines[0].trigger.artifacts.addAll([[name: "a", version: "1"],  [name: "a"]])
-    pipelines[1].trigger.artifacts.addAll([[name: "a"], [name: "a", version: "1"]])
+    pipelines[0].trigger.artifacts.addAll([Artifact.builder().name("a").version("1").build(), Artifact.builder().name("a").build()])
+    pipelines[1].trigger.artifacts.addAll([Artifact.builder().name("a").build(), Artifact.builder().name("a").version("1").build()])
```

Comment:
@ezimanyi I had to change this test following your recent change to `Artifact` in Kork. Frankly I'm baffled at how it ever worked.

There's also a remaining failing test in `ManifestEvaluatorTest.artifactManifestNoAccountFailure`. That also seems to be related to artifact changes. Any chance you could take a look?

Reply:
Thanks for that change!  I have no idea how that was working...a few months ago I removed the public all-arg and no-arg constructors for Artifact which is what should have broken that test (and I fixed the tests at that time).  Maybe this wasn't running somehow.  Either way, thanks for fixing it.

Looking at the `ManifestEvaluatorTest` now.

Reply:
Thanks

Reply:
The failure is due to an overzealous null-to-empty conversion that I did which I reverted in `kork` 7.43.1.  Changing this to bump `kork` to 7.43.1 instead of 7.43.0 will fix the test.

Reply:
Coolio. I'll do that. Thanks again.